libavcodec/jpeg2000dec.c

```diff
@@ -479,6 +479,24 @@ static int get_siz(Jpeg2000DecoderContext *s)
     return 0;
 }
 
+static int get_cap(Jpeg2000DecoderContext *s, int len)
+{
+    av_log(s->avctx, AV_LOG_ERROR, "Parsing of CAP segment not implemented\n");
+    av_log(s->avctx,AV_LOG_ERROR,"Skipping %d bytes of the CAP segment\n" ,len-2);
```

Comment:
```suggestion
    av_log(s->avctx, AV_LOG_INFO, "Skipping %d bytes of the CAP segment\n" , len-2);
```

Combine with the previous `av_log()`.

I have not seen examples of multi-line `av_log()` being split into multiple calls to `av_log()`.